django/contrib/postgres/fields/ranges.py

```diff
@@ -132,6 +133,45 @@ def db_type(self, connection):
 RangeField.register_lookup(lookups.Overlap)
 
 
+class DateTimeRangeContains(models.Lookup):
+    lookup_name = 'contains'
+    internal_type = 'DateTimeField'
+
+    def as_sql(self, compiler, connection):
+        self._pre_process_rhs(compiler)
```

Comment:
I'm not sure if the additional methods are helping with readability. 1 method with some comments explaining each block might be simpler.